cedar-policy-core/src/ast/policy_set.rs

```diff
@@ -182,7 +182,7 @@ impl PolicySet {
         template_id: PolicyID,
         new_id: PolicyID,
         values: HashMap<SlotId, EntityUID>,
-    ) -> Result<(), LinkingError> {
+    ) -> Result<&Policy, LinkingError> {
```

Comment:
Since we're now returning a value, can we document what we're returning in the docstring?  (It's probably obvious that it returns the new linked policy, but good to say so explicitly anyhow)